website/blog/2024-06-24-run-ferretdb-cloudnativepg-kubernetes.md

```diff
@@ -0,0 +1,410 @@
+ferretdb>
+```
+
+### Test CRUD commands in `mongosh`
+
+Now that you are connected to the FerretDB instance, let's run some CRUD commands.
+
+Start by creating a collection and inserting some documents:
+
+```js
+db.testCollection.insertMany([
+  { name: 'Alice', age: 25, city: 'Wonderland' },
+  { name: 'Bob', age: 30, city: 'Builderland' },
+  { name: 'Charlie', age: 35, city: 'Chocolate Factory' }
+])
+```
+
+Output:
+
+```json5
+{
+  acknowledged: true,
+  insertedIds: {
+    '0': ObjectId('66741445f0f21e98f96bade8'),
+    '1': ObjectId('66741445f0f21e98f96bade9'),
+    '2': ObjectId('66741445f0f21e98f96badea')
+  }
+}
+```
+
+Next let's update a document:
+
+```js
+db.testCollection.updateOne({ name: 'Alice' }, { $set: { age: 26 } })
+```
+
+Output:
+
+```json5
+{
+  acknowledged: true,
+  insertedId: null,
+  matchedCount: 1,
+  modifiedCount: 1,
+  upsertedCount: 0
+}
+```
+
+Delete documents where age is less than 35:
+
+```js
+db.testCollection.deleteMany({ age: { $lt: 35 } })
+```
+
+Output:
+
+```json5
+{ acknowledged: true, deletedCount: 2 }
+```
+
+Count the number of documents in the collection:
+
+```js
+db.testCollection.countDocuments()
+```
+
+Output:
+
+```json5
+1
+```
+
+### Connect to Postgres cluster via `psql`
+
+Want to see what the data looks like in Postgres?
+
+From another terminal window, let's create a temporary `postgres-client` pod to connect to the `postgres-cluster-1` pod.
+
+```sh
+kubectl exec -it postgres-cluster-1 -n cnpg -- /bin/bash
+```
+
+Connect to the PostgreSQL database:
+
+```sh
+psql postgresql://ferretuser:<password>@postgres-cluster-rw.cnpg.svc.cluster.local:5432/ferretdb
+```
+
+```text
+postgres@
+psql postgresql://ferretuser:<password>@postgres-cluster-rw.cnpg.svc.cluster.local:5432/ferretdb
+psql (16.2 (Debian 16.2-1.pgdg110+2))
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
+Type "help" for help.
+
+ferretdb=> SET SEARCH_PATH TO FERRETDB;
+SET
+ferretdb=> \dt
+                      List of relations
+  Schema  |            Name             | Type  |   Owner
+----------+-----------------------------+-------+------------
+ ferretdb | _ferretdb_database_metadata | table | ferretuser
+ ferretdb | testcollection_c141f891     | table | ferretuser
+(2 rows)
+ferretdb=> select * from testcollection_c141f891;
+                                                                                                                        _jsonb
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ {"$s": {"p": {"_id": {"t": "objectId"}, "age": {"t": "int"}, "city": {"t": "string"}, "name": {"t": "string"}}, "$k": ["_id", "name", "age", "city"]}, "_id": "66741445f0f21e98f96badea", "age": 35, "city": "Chocolate Factory", "name": "Charlie"}
+(1 row)
+```
+
+## Clean up resources
+
```

Comment:
Maybe also a quick explanation message might be added. Something like, "when we don't need FerretDB nor Postgres cluster anymore, we can just cleanup the whole environment completely". 

Btw, does `minikube delete` delets the whole cluster? do we need to delete a particular namespace if we are going to delete the whole cluster?